src/steps/03-Background/CurrentEnvironment/InstanceDetails.vue

```diff
@@ -525,6 +527,36 @@ export default class InstanceDetails extends Mixins(SaveOnLeave) {
     return isValid;
   }
 
+  /**
+   * Compiles and returns the sequence number for pricing details section
+   */
+  getPricingDetailsSequenceNum(): string {
+    if (this.hasTellUsAboutInstanceHeading) {
+      return "4.";
+    } else {
+      return "3.";
+    }
```

Comment:
please use js ternaries for the above and the 2 in the method below - can be done in 1 line and is more inline with our codebase.
`return this.hasTellUsAboutInstanceHeading ? "4." : "3.";`